crates/ruma-common/src/events/enums.rs

```diff
@@ -67,6 +67,9 @@ event_enum! {
         "m.room.encrypted" => super::room::encrypted,
         "m.room.message" => super::room::message,
         "m.room.message.feedback" => super::room::message::feedback,
+        // This attribute is necessary because of the import of `SyncRoomRedactionEvent`.
+        // See: https://github.com/rust-lang/rust/issues/96698
+        #[allow(unused_qualifications)]
```

Comment:
I was thinking of putting this attribute in the macro code. Is that not possible, or do you prefer this solution?

Reply:
Oh right, I'll do that then.